### ISSUE TITLE
Continuous anchor face restrictions do not work in v1.7.5

### ISSUE THREAD

**Issue by Domiii** [sporritt/jsPlumb#384]
As explained [here](http://stackoverflow.com/questions/30690294/continuous-anchor-face-restrictions-do-not-work-in-jsplumb-1-7-5), restrictions on `Continuous` anchor faces  (as explained in the docs [here](http://www.jsplumb.org/doc/anchors.html#continuous)) are ignored in v1.7.5. It still spreads the connections all over the place.

Please advice!
Maybe you can tell me in which version this still worked?

**Comment by sporritt** [comment]
if i try this in the statemachine demo:

```
instance.makeSource(windows, {
            filter: ".ep",
            anchor: ["Continuous", { faces:["top"] }],
            connector: [ "StateMachine", { curviness: 20 } ],
            connectorStyle: { strokeStyle: "#5c96bc", lineWidth: 2, outlineColor: "transparent", outlineWidth: 4 },
            maxConnections: 5,
            onMaxConnections: function (info, e) {
                alert("Maximum connections (" + info.maxConnections + ") reached");
            }
        });
```

it chooses the top edge only.  maybe you can supply a jsfiddle or something as a test case?

**Comment by Domiii** [comment]
Ah Ok! I finally found the culprit! The `type` I used in my `connect` call defined `anchor: 'Continuous'` which overrode the per-connect options. [This is a working fiddle](http://jsfiddle.net/gXuS7/549/).

Thanks for the help!

It would be great if all possible sources of endpoint settings and their corresponding priorities can be explained in the documentation? E.g.:

"When looking for endpoints a new connection will first consider `type` parameters, then source and target parameters, then ....... ???????" :)
